src/stories/common/BaseButton.stories.js

```diff
@@ -17,7 +17,7 @@ const Template = (args) => ({
     setup() {
         return { args };
     },
-    template: '<base-button v-bind="args"> Test Button </base-button>',
+    template: '<base-button v-bind="args">Test Button</base-button>',
```

Comment:
Why was this changed? If you accidently change other files try not to commit the changes.